api/bases/test.openstack.org_tempests.yaml

```diff
@@ -133,6 +133,113 @@ spec:
                     description: WorkerFile is the detailed concurry spec file
                     type: string
                 type: object
+              tempestconfRun:
+                description: TempestSpec PythonTempestconf parts
+                properties:
+                  append:
+                    default: ""
+                    description: Append values to tempest.conf
+                    type: string
+                  collectTiming:
+                    default: false
+                    description: Collect per-API call timing information.
+                    type: boolean
+                  convertToRaw:
+                    default: false
+                    description: Convert images to raw format before uploading.
+                    type: boolean
+                  create:
+                    default: true
+                    description: Create Tempest resources
+                    type: boolean
+                  createAccountsFile:
+                    default: ""
+                    description: Generate Tempest accounts file.
+                    type: string
+                  debug:
+                    default: false
+                    description: Print debugging information.
+                    type: boolean
+                  deployerInput:
+                    default: ""
+                    description: Path to deployer file
```

Comment:
How you are going to map the deployer input into the container ?
Should we have predefined location and in case it present use it ?

Reply:
I think we've already discussed this. I'm adding it here for reference. The deployerInput is mapped into the tempest pod as a file using ConfigMap - [here](https://github.com/openstack-k8s-operators/test-operator/pull/10/commits/e9148516cda9fd11e8a180e5a799c2dbedc9ea42#diff-19fb6c8a471eb8f571e5e959d274704a582df8fa55a60f500593fcba5efb8823R421). The location of the file is set by the test-operator as well one line below.